ts/Core/Axis/AxisDefaults.ts

```diff
@@ -1538,15 +1538,29 @@ namespace AxisDefaults {
 
         /**
          * Additional range on the right side of the xAxis. Works similar to
-         * `xAxis.maxPadding`, but value is set in milliseconds. This property
-         * should be set to the same value for both the main `xAxis` and the
+         * `xAxis.maxPadding`, but value is set in milliseconds, percentage or pixels.
```

Comment:
This doclet mentions milliseconds, which is true for datetime axes (as is the case in Stock). But the more precise term is _axis values_. It may be better to use that term in case this works with linear axes too, now or in the future.

> Works similar to `xAxis.minPadding`, but the value is set in terms of axis values, percentage or pixels.